tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -97,6 +97,28 @@ public function testListProject($data):array
         $this->assertEquals($id, $response['body']['projects'][0]['$id']);
         $this->assertEquals('Project Test', $response['body']['projects'][0]['name']);
 
+        $response = $this->client->call(Client::METHOD_GET, '/projects', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders(), [
+            'search' => $id
+        ]));
+
+        $this->assertEquals(200, $response['headers']['status-code']);
+        $this->assertNotEmpty($response['body']);
+        $this->assertEquals('Project Test', $response['body']['projects'][0]['name']);
```

Comment:
Can we keep these tests consistent? I really liked this format on listFunctions - the assertions are more strict. 

```
        $this->assertEquals($function['headers']['status-code'], 200);
        $this->assertEquals($function['body']['sum'], 1);
        $this->assertIsArray($function['body']['tags']);
        $this->assertCount(1, $function['body']['tags']);
        $this->assertEquals($function['body']['tags'][0]['$id'], $data['tagId']);
```

Reply:
Updated both usages in this function.